concordium-sdk/src/main/java/com/concordium/sdk/Client.java

```diff
@@ -303,6 +304,28 @@ public PeerStatistics getPeerStatistics(final boolean includeBootstrappers) {
         return PeerStatistics.parse(value);
     }
 
+    /**
+     * Get the IDs of the bakers registered in the given block.
+     * Note that this list is in general different from the bakers that are
+     * returned as part of [get_birk_parameters](Client::get_birk_parameters).
```

Comment:
Please convert to proper javadoc. Also 'get birk parameters' is not currently in the branch?

Reply:
done